### DISCUSSION TITLE
Doesn't work for Dutch language

### DISCUSSION THREAD

**Discussion by Tauvic** [DerwenAI/pytextrank#245] [Q&A] [answered]
import spacy
nlp = spacy.load('nl_core_news_lg')
nlp.add_pipe("textrank")

ValueError: [E002] Can't find factory for 'textrank' for language Dutch (nl). This usually happens when spaCy calls `nlp.create_pipe` with a custom component name that's not registered on the current language class. If you're using a Transformer, make sure to install 'spacy-transformers'. If you're using a custom component, make sure you've added the decorator `@Language.component` (for function components) or `@Language.factory` (for class components).

**Comment by 0dB** [comment, marked as answer]
It looks like you are missing
`import pytextrank`
? When I add that it works for me.
